### PR TITLE
[ON HOLD] Generate extra DAppNode packages JSON

### PR DESCRIPTION
**Pull request by 3alpha**
Generates JSON with packages to be installed at onboarding.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -98,9 +102,9 @@ jobs:
           ls -lrt images/DAppNode-debian-bullseye-amd64.iso
 
       # Set new name for the release asset
-      - name: Set DAppNode unttended ISO name
+      - name: Set DAppNode unattended ISO name
         run: |
-          cp ./images/DAppNode-debian-bullseye-amd64.iso DAppNode-${CORE_VERSION}-debian-bullseye-amd64-unattended.iso
+          cp ./images/DAppNode-debian-bullseye-amd64.iso DAppNode-${CORE_VERSION}-${EXTRA_PKGS}-debian-bullseye-amd64-unattended.iso
```
Comment (pablomendezroyo) [inline .github/workflows/release.yml:107]: You are overwriting the standard release, instead you should create a new step to create the ISO with the extra packages so its a new asset to be added to the release

File: iso/scripts/extra_dappnode_pkgs/testpkg1 [hunk 2]
```diff
@@ -0,0 +1,4 @@
+DAppNodePackage Test 1
```
Comment (pablomendezroyo) [inline iso/scripts/extra_dappnode_pkgs/testpkg1:1]: these testpkgx can be remoed right?

**Review by pablomendezroyo — CHANGES_REQUESTED** [review]

File: iso/scripts/extra_dappnode_pkgs/testpkg1 (hunk 2 above)
Comment (pablomendezroyo) [reply iso/scripts/extra_dappnode_pkgs/testpkg1:1]: Nevermind

File: .github/workflows/release.yml (hunk 1 above)
Comment (pablomendezroyo) [reply .github/workflows/release.yml:107]: Edit compose env on the fly

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -152,12 +156,20 @@ jobs:
...
-          cat CHANGELOG.md
         env:
           SHASUM_ATTENDED: ${{ steps.shasum-attended.outputs.SHASUM_ATTENDED }}
           SHASUM_UNATTENDED: ${{ steps.shasum-unattended.outputs.SHASUM_UNATTENDED }}
-          SHASUM_EXTRA_PKGS: ${{ steps.shasum-unattended.outputs.SHASUM_EXTRA_PKGS }}
 
+      - name: Write release content for extra pkgs
+        if: ${{ github.event.inputs.EXTRA_PKGS_CI != '' }}
+        run: |
+          echo -en "# Unattended version with extra packages: ${EXTRA_PKGS_CI}\nInstall DAppNode easily using the ISO: **DAppNode-${CORE_VERSION}-${EXTRA_PKGS_CI}-debian-bullseye-amd64-unattended.iso**\nDo a reboot right after the installation\n:warning: **Warning**: This ISO will install dappnode automatically, deleting all existing partitions in the disk\n\ndefault login data:\n - **__user__**: dappnode\n - **__password__**: dappnode.s0\n## ISO SHA-256 Checksum\n\`\`\`\nshasum -a 256 DAppNode-${CORE_VERSION}-${EXTRA_PKGS_CI}-debian-bullseye-amd64-unattended.iso\n${SHASUM_EXTRA_PKGS}\n\`\`\`\n" >> CHANGELOG.md
```
Comment (pablomendezroyo) [inline .github/workflows/release.yml:166]: This markdown should be included in the above step, why adding a new step for a new changelog?

**Review by pablomendezroyo — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 3 above)
Comment (3alpha) [reply .github/workflows/release.yml:166]: Because this should be printed only if extra packages are selected.

Comment (pablomendezroyo) [reply .github/workflows/release.yml:166]: Right!

**Review by pablomendezroyo — DISMISSED** [review]

**Comment by pablomendezroyo** [comment]
@3alpha what is the current status of this?

**Comment by 3alpha** [comment]
It is done, however it is missing dappmanger part which it will make it work.

**Pull request closed without merging**